erts/etc/common/erlc.c

```diff
@@ -908,7 +913,13 @@ safe_realpath(char* file)
      * Solaris.
      */
     char* real_name = emalloc(PATH_MAX + 1);
-    return realpath(file, real_name);
+    char* result = realpath(file, real_name);
+    if (result == NULL) {
+        free(real_name);
+    }
+    /* codechecker_false_positive [Malloc] we are not leaking
+       real_name because result == real_name if result != NULL */
```

Comment:
To avoid codechecker specific suppression
could this be fixed by changing to
if (result != real_name)

Reply:
Thanks, I will give it a try.

Reply:
I have updated the code (we will see if it works after our code checker build has executed).